assets/queries/ansible/aws/aws_sns_topic_subscription/query.rego

```diff
@@ -0,0 +1,44 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  snsTopicCommunity := task["community.aws.sns_topic"]
+  snsTopicName := task.name
+  object.get(snsTopicCommunity, "subscriptions", "undefined") != "undefined"
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.sns_topic}}", [snsTopicName]),
+                "issueType":        "MissingAttribute",
+                "keyExpectedValue": "community.aws.redshift.encrypted should be undefined",
+                "keyActualValue":   "community.aws.redshift.encrypted is defined"
+              }
+}
+
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  snsTopic := task["sns_topic"]
+  snsTopicName := task.name
+  object.get(snsTopic, "subscriptions", "undefined") != "undefined"
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{sns_topic}}", [snsTopicName]),
+                "issueType":        "MissingAttribute",
+                "keyExpectedValue": "community.aws.redshift.encrypted should be undefined",
+                "keyActualValue":   "community.aws.redshift.encrypted is defined"
```

Comment:
Change to:
                "keyExpectedValue": "sns_topic.subscriptions should be undefined",
                "keyActualValue":   "sns_topic.subscriptions is defined"